python/orca/src/bigdl/orca/learn/tf/tf_estimator.py

```diff
@@ -14,22 +14,27 @@
 # limitations under the License.
 #
 
-from bigdl.orca.ray import RayContext
-from bigdl.orca.learn.tf.tf_runner import TFRunner
 import ray
+
+from bigdl.orca.ray import OrcaRayContext
+from bigdl.orca.learn.tf.tf_runner import TFRunner
 from bigdl.orca.learn.dl_cluster import RayDLCluster
+from typing import TYPE_CHECKING, Any
+if TYPE_CHECKING:
+    from typing import Dict, Optional, Callable
```

Comment:
put `Any ` under `if TYPE_CHECKING` or move line24 out

Reply:
if I put `Any` under `if TYPE_CHECKING`, UT will report error: `"no module named Any."`

Reply:
What if I just remove line 24 and add it in line 22?

Reply:
> What if I just remove line 24 and add it in line 22?

That sounds good!